pkg/slack/events/messages/message_handler.go

```diff
@@ -97,59 +103,428 @@ func postResponse(client *slack.Client, event *slackevents.MessageEvent, respons
 	return nil
 }
 
-func help(client *slack.Client, event *slackevents.MessageEvent, botCommands []parser.BotCommand, allowPrivate bool) {
-	helpMessage := ""
-	for _, command := range botCommands {
+// GenerateHelpOverviewMessage creates the help overview message content
+func GenerateHelpOverviewMessage(allowPrivate bool) string {
+	helpMessage := "*🤖 Cluster Bot - Quick Start*\n\n"
+
+	// Common commands
+	helpMessage += "*Most Used Commands:*\n"
+	helpMessage += "• `help launch` - Launch OpenShift clusters\n"
+	helpMessage += "• `help rosa` - ROSA (Red Hat OpenShift Service on AWS)\n"
+	helpMessage += "• `list` - See active clusters\n"
+	helpMessage += "• `done` - Terminate your cluster\n"
+	helpMessage += "• `auth` - Get cluster credentials\n\n"
+
+	// All available commands with detailed usage
+	helpMessage += "\n*All Commands:*\n"
+
+	helpMessage += "\n**Cluster Launching:**\n"
```

Comment:
These double stars `**` are not translating into anything valid in slack:
<img width="701" height="607" alt="image" src="https://github.com/user-attachments/assets/9b00badb-b347-42ab-a813-b3da5d3eebe6" />

Reply:
pushed new changes